server/searchPage.js

```diff
@@ -10,9 +10,12 @@ const pool = new Pool({
 
 
 module.exports.applySearch = async event => {
-		// const selectAllCleaners = "SELECT * FROM cleaner_profile"
+		/* @Gurleen: This is a SELECT statement to help with the filter story
```

Comment:
Thanks for doing this, this will help later tickets greatly - lets merge this PR ASAP so its deployed to the team